modules/microshift-adding-app-rpms-to-blueprint.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embedding-apps-tutorial.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-adding-app-rpms-to-blueprint_{context}"]
+= Adding application RPMs to a blueprint
+
+To add application RPMs to a blueprint, you must create a local repository that Image Builder can use to create the ISO. With this procedure, the required container images for your workload can be pulled over the network.
+
+.Prerequisites
+
+* You have root access to the host.
+* Workload or application RPMs exist in the `~/rpmbuild/RPMS` directory.
+
+.Procedure
+
+. Create a local RPM repository by running the following command:
++
+[source,terminal]
+----
+$ createrepo ~/rpmbuild/RPMS/
+----
+
+. Give Image Builder access to the RPM repository by running the following command:
++
+[source,terminal]
+----
+$ sudo chmod a+rx ~
+----
++
+[NOTE]
+====
+You must ensure that Image Builder has all of the necessary permissions to access all of the files needed for image building, or the build cannot proceed.
+====
++
+. Create the blueprint file, `repo-local-rpmbuild.toml` using the following template:
++
+[source,terminal]
+----
+id = "local-rpm-build"
+name = "RPMs build locally"
+type = "yum-baseurl"
+url = "file://<PATH>/rpmbuild/RPMS" <1>
+check_gpg = false
+check_ssl = false
+system = false
+----
+<1> Specify part of the path to create a location that you choose. Use this path in the later commands to set up the repository and copy the RPMs.
+
+. Add the repository as a source for Image Builder by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli sources add repo-local-rpmbuild.toml
+----
+
+. Add the RPM to your `.toml` blueprint file, by adding the following lines:
```

Comment:
According to the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#blueprint), you can just use blueprint. And similar to YAML files/format, we use TOML.